xrpl4j-client/src/main/java/org/xrpl/xrpl4j/client/XrplClient.java

```diff
@@ -127,6 +130,30 @@ public XrplClient(final HttpUrl rippledUrl) {
     this(JsonRpcClient.construct(rippledUrl));
   }
 
+  /**
+   * Public constructor that allows for configuration of connect and read timeouts.
+   *
+   * @param rippledUrl         The {@link HttpUrl} of the node to connect to.
+   * @param connectTimeout     A {@code long} indicating the client's connect timeout.
+   * @param connectTimeoutUnit The {@link TimeUnit} that {@code connectTimeout} is denominated in.
+   * @param readTimeout        A {@code long} indicating the client's read timeout.
+   * @param readTimeoutUnit    The {@link TimeUnit} that {@code readTimeout} is denominated in.
+   */
+  public XrplClient(
+    HttpUrl rippledUrl,
+    long connectTimeout,
```

Comment:
Any reason not use two `java.time.Duration` parameters here instead of the current four?

If that's easy, I prefer it. However, we could always add an override constructor later if you want this one to merge quickly (esp since this class is marked `@Beta`).

Reply:
This is resolved.